build.sbt

```diff
@@ -438,6 +438,7 @@ def SbtPluginPlugins  = AutomateHeaderPlugin && BintrayPlugin && PluginsAccessor
 
 lazy val api = (project in file("service/core/api"))
   .settings(runtimeLibCommon: _*)
+  .settings(forkedTests: _*) // needed because of parser-combinators -> https://github.com/scala/scala-parser-combinators/issues/197
```

Comment:
This seems to be needed now. Play upgraded the parser-combinator to 1.1.2 and that doesn't play well when building with Scala 2.11. 

Therefore I tracked all modules that were depending on parser-combinator and added the forked tests settings. 

See https://github.com/scala/scala-parser-combinators/issues/197

Reply:
An alternative to that is updating to sbt 1.3.x, with the requirements that brings...  Despite said requirements, I'd go with that over migrating the build setup to forking the tests.